community/cypher/cypher-planner-3.4/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/AstRewritingTestSupport.scala

```diff
@@ -22,6 +22,50 @@ package org.neo4j.cypher.internal.compiler.v3_4.planner
 import org.neo4j.cypher.internal.compiler.v3_4.parser.ParserFixture
 import org.neo4j.cypher.internal.frontend.v3_4.ast.AstConstructionTestSupport
 import org.neo4j.cypher.internal.util.v3_4.test_helpers.CypherTestSupport
+import org.neo4j.cypher.internal.ir.v3_4.PlannerQuery
+import org.neo4j.cypher.internal.planner.v3_4.spi.PlanningAttributes.{Cardinalities, ReadOnlies, Solveds}
+import org.neo4j.cypher.internal.util.v3_4.Cardinality
+import org.neo4j.cypher.internal.util.v3_4.attribution.{Id, SequentialIdGen}
+import scala.language.implicitConversions
+
+trait LogicalPlanConstructionTestSupport extends CypherTestSupport {
+  implicit val idGen = new SequentialIdGen()
+
+  implicit protected def idName(name: String): String = name
+
+  implicit protected def idSymbol(name: Symbol): String = name.name
+
+  class FakeSolveds extends Solveds {
```

Comment:
These kinds of of test classes are usually prefixed with `Stub...` rather than `Fake...`. I think this could be changed because it makes naming more consistent.